.github/workflows/ruby.yml

```diff
@@ -6,13 +6,16 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6', '3.0' ]
```

Comment:
At the risk of uncovering a surprise and painful bug, what do you think about adding 2.5 and 2.7 "support" here?

Reply:
@oreoshake 2.5 EOLs at the end of this month, so it doesn't make a ton of sense. 2.7 makes sense since it is the last in the 2.x series, so I added it.